packages/contracts/contracts/core/component/Permissions.sol

```diff
@@ -5,29 +5,26 @@
 pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
-import "@opengsn/contracts/src/BaseRelayRecipient.sol";
-import "@opengsn/contracts/src/interfaces/IRelayRecipient.sol";
-
+import "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
 import "./../IDAO.sol";
 import "./../acl/ACL.sol";
 
-/// @dev Used to silence compiler warning in order to call trustedForwarder() on the DAO
-interface Relay {
-    function trustedForwarder() external view returns (address);
-}
```

Comment:
Was it ok to remove this @nivida @novaknole? I don't really know what this was for.

Reply:
i see @Michael-A-Heuer updated OZ,
and also i see he is not using trustedForwarder() anymore
so removing this interface should be fine.
just in case can u confirm @novaknole ?